Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md

```diff
@@ -851,7 +864,58 @@ Import H2O’s training and test subsets of the Retail Store Forecasting dataset
 
 ![retail-store-train-test-datasets](assets/retail-store-train-test-datasets.jpg)
 
-## Appendix B: Time Series FAQ
+
+## Appendix B: Forecasting Outside of the Forecast Horizon Tips 
+
+### Time Series Forecasting with Test Time Augmentation
+
+When you set the experiment’s forecast horizon, you are telling the Driverless AI experiment the dates this model will be asked to forecast for. In the Walmart Sales example, we set the Driverless AI forecast horizon to 1 (1 week in the future). This means that Driverless AI expects this model to be used to forecast 1 week after training ends. Since the training data ends on 2020-10-26, then this model should be used to score for the week of 2020-11-02.
+
+What should the user do once the 2020-11-02 week has passed?
+
+There are two options:
+
+Option 1: Trigger a Driverless AI experiment to be trained once the forecast horizon ends. A Driverless AI experiment will need to be re-trained every week.
+
+Option 2: Use Test Time Augmentation to update historical features so that we can use the same model to forecast outside of the forecast horizon.
```

Comment:
@Castro38 I think it will be easier on the eyes to have "option 2" be bold

Reply:
I agree, DONE.